### PR TITLE
Added Azure security basics

### PR DESCRIPTION
**Pull request by nprokopic**
Added basic information about encryption of local and persistent storage for Azure-based installations.

### CODE REVIEW

File: src/content/basics/security/index.md [hunk 1]
```diff
@@ -35,3 +35,14 @@ Non-persistent volumes as well as docker images and logs are stored under `/var/
 
 Persistent storage is managed by the `StorageClass` resource in Kubernetes. By default, the `StorageClass` resource is provided as an Elastic Block Storage (EBS) volumes. These volumes are encrypted via AWS EBS Encryption. The key is created, stored and deleted using AWS Key Management Service (KMS).
 
+### Azure
+
+This section applies to Azure-based installations only.
+
+#### Encryption of local storage {#azure-local-storage}
+
+Kubelet data is stored under `/var/lib/kubelet`, Docker images under `/var/lib/docker` and for master nodes etcd data is stored under `/var/lib/etcd`. On Azure, `/var/lib/kubelet`, `/var/lib/docker` and `/var/lib/etcd` are Azure Managed disks (Premium SSD). Azure managed disk automatically encrypts data by default with platform-managed keys (managed by Microsoft). See more details in [Azure managed disk documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disk-encryption).
```
Comment (JosephSalisbury) [inline src/content/basics/security/index.md]:
```suggestion
kubelet data is stored under `/var/lib/kubelet`, Docker images under `/var/lib/docker` and for master nodes etcd data is stored under `/var/lib/etcd`. On Azure, `/var/lib/kubelet`, `/var/lib/docker` and `/var/lib/etcd` are Azure Managed Disks (Premium SSD). Azure Managed Disks are automatically encrypted by default with platform-managed keys (managed by Microsoft). See more details in [Azure managed disk documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disk-encryption).
```

File: src/content/basics/security/index.md [hunk 2]
```diff
@@ -35,3 +35,14 @@ Non-persistent volumes as well as docker images and logs are stored under `/var/
...
+
+#### Encryption of persistent storage {#azure-persistent-storage}
+
+Persistence storage is managed by the `StorageClass` resource in Kubernetes. By default, storage class is provided by Azure Managed disks (Premium SSD). Azure managed disk automatically encrypts data by default with platform-managed keys (managed by Microsoft). See more details in [Azure managed disk documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disk-encryption).
```
Comment (JosephSalisbury) [inline src/content/basics/security/index.md]:
```suggestion
Persistent storage is managed by the `StorageClass` resource in Kubernetes. The default Kubernetes Storage Class is backed by Azure Managed Disks (Premium SSD). Azure Managed Disks are automatically encrypted by default with platform-managed keys (managed by Microsoft). See more details in [Azure managed disk documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disk-encryption).
```

**Comment by nprokopic** [comment]
pinging @giantswarm/sig-content could you folks check and optionally suggest better working?

**Comment by Oshratn** [comment]
I'm good with the repetition since it is different types of storage and if a reader scans rather than reads s/he will be looking for headlines and then reading it.

**Pull request merged**